monai/transforms/croppad/array.py

```diff
@@ -404,13 +422,14 @@ def __call__(self, img: torch.Tensor, slices: Tuple[slice, ...]) -> torch.Tensor
         slices = tuple([slice(None)] + slices_[:sd])
 
         img_t: MetaTensor = convert_to_tensor(data=img, track_meta=get_track_meta())
+        _orig_size = img_t.shape[1:]
         img_t = img_t[slices]  # type: ignore
         if get_track_meta():
             self.update_meta(tensor=img_t, slices=slices)
             cropped_from_start = np.asarray([s.indices(o)[0] for s, o in zip(slices[1:], orig_size)])
             cropped_from_end = np.asarray(orig_size) - img_t.shape[1:] - cropped_from_start
             cropped = list(chain(*zip(cropped_from_start.tolist(), cropped_from_end.tolist())))
-            self.push_transform(img_t, extra_info={"cropped": cropped})
+            self.push_transform(img_t, orig_size=_orig_size, extra_info={"cropped": cropped})
```

Comment:
I see you added `orig_size=orig_size` for all the crop transforms, so the previous inverse logic are wrong?

Thanks.

Reply:
previously there's no `orig_size` info for some of the cropping, I think it's useful to track the previous size in some cases, given that we don't inverse the multi sample cropping.